src/dev/build/args.ts

```diff
@@ -37,6 +37,7 @@ export function readCliArgs(argv: string[]) {
       'debug',
       'all-platforms',
       'example-plugins',
+      'download-only',
```

Comment:
I feel like `--download-only` is a pretty generic flag that could apply to many downloads done during the build. Maybe we should make this more chromium specific?

Reply:
:thu: I may have gotten too far ahead on planning.  

We have downloads for native modules, node, EPR, and beats and I started looking into separating download from installation, in case we wanted to cache it all on our agents.  Doesn't seem high priority, and not sure when I'll get to it.  Will update the name in a moment.